recipes/gtest/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.7.0" %}
+{% set sha256 = "f73a6546fdf9fce9ff93a5015e0333a8af3062a152a9ad6bcb772c96687016cc" %}
+
+package:
+  name: gtest
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/googletest/archive/release-{{ version }}.tar.gz
+  fn: release-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - cmake
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgtest.a  # [unix]
```

Comment:
Added a test for `libgtest_main.a` and `include/gtest` directory. No windows test for now.